docs/source/configuration/persisted-queries.mdx

```diff
@@ -0,0 +1,183 @@
+---
+title: Safelisting with persisted queries
+description: Reject unrecognized operations while minimizing request latency
+minVersion: 1.25.0
+---
+
+<blockquote>
+
+**This feature is available only with a [GraphOS Enterprise plan](/graphos/enterprise/).** It is currently in [preview](/resources/product-launch-stages#preview).
+
+If your organization _doesn't_ currently have an Enterprise plan, you can test this functionality by signing up for a free [Enterprise trial](/graphos/org/plans/#enterprise-trials).
+
+</blockquote>
+
+With [GraphOS Enterprise](/graphos/enterprise/), you can enhance your supergraph's security by maintaining a **persisted query list** (**PQL**) for your supergraph's self-hosted router. The Apollo Router checks incoming requests against the PQL, a safelist of trusted operations made by your first-party apps.
+
+```mermaid
+flowchart LR
+  subgraph Apps["First-party apps"]
+  direction LR
+  webClient("Web client");
+  androidClient("Android client");
+  iosClient("iOS client");
+  end
+
+    subgraph Router["Apollo Router"]
+        direction TB
+        router(["Graph<br/>Router"]);
+        pql[[Persisted<br/>Query List]];
+        router <--> pql
+    end
+
+  subgraph Supergraph
+  direction LR
+  api1(["GraphQL API"]);
+  api2(["GraphQL API"]);
+  api3(["GraphQL API"]);
+
+  end
+  Apps -->|Queries| Router -->|Trusted<br/> queries| Supergraph
+```
+
+Your router can use its persisted query list (PQL) to both **protect your supergraph** and **speed up your clients' operations**:
+
+- When you enable **safelisting**, your router _rejects_ any incoming operations not registered in its PQL.
+- Client apps can execute an operation by providing its PQL-specified ID _instead of_ an entire operation string.
+    - Querying by ID can significantly reduce latency and bandwidth usage for very large operation strings.
+    - Your router can _require_ that clients provide operations by ID and _reject_ full operation strings (even operation strings present in the PQL).
+
+## Differences from automated persisted queries
+
+The Apollo Router supports a related feature called [automatic persisted queries](./in-memory-caching#caching-automatic-persisted-queries-apq) (APQ). With APQ, clients can execute a GraphQL operation by sending the SHA256 hash of its operation string instead of the entire string. **APQ doesn't support safelisting** because the router dynamically populates its APQ cache over time with _any_ operations it receives.
```

Comment:
```suggestion
The Apollo Router also supports a related feature called [automatic persisted queries](./in-memory-caching#caching-automatic-persisted-queries-apq) (APQ). With APQ, clients can execute a GraphQL operation by sending the SHA256 hash of its operation string instead of the entire string. **APQ doesn't support safelisting** because the router dynamically populates its APQ cache over time with _any_ operations it receives.
```

(as in https://github.com/apollographql/docs/pull/496)